datasets/esa-worldcover/collection/template.json

```diff
@@ -0,0 +1,268 @@
+            "type": "application/pdf",
+            "title": "2021 Product Version 2.0.0 Validation Report"
+        }
+    ],
+    "stac_extensions": [
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/raster/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/classification/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/table/v1.2.0/schema.json"
+    ],
+    "keywords": [
+        "Global",
+        "Land Cover",
+        "Sentinel",
+        "ESA"
+    ],
+    "msft:short_description": "Global land cover product at 10 meter resolution based on Sentinel-1 and Sentinel-2 data",
+    "msft:storage_account": "ai4edataeuwest",
+    "msft:container": "esa-worldcover",
+    "providers": [
+        {
+            "name": "ESA",
+            "roles": [
+                "licensor",
+                "producer"
+            ],
+            "url": "https://esa-worldcover.org"
+        },
+        {
+            "name": "ESA WorldCover Consortium",
+            "description": "The WorldCover product is developed by a consortium led by VITO Remote Sensing together with partners Brockmann Consult, CS SI, Gamma Remote Sensing AG, IIASA and Wageningen University",
+            "roles": [
+                "processor"
+            ],
+            "url": "https://esa-worldcover.org"
+        },
+        {
+            "name": "Microsoft",
+            "roles": [
+                "host"
+            ],
+            "url": "https://planetarycomputer.microsoft.com"
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "ESA WorldCover thumbnail",
+            "href": "https://ai4edatasetspublicassets.blob.core.windows.net/assets/pc_thumbnails/esa-worldcover-thumb.png",
+            "type": "image/png",
+            "roles": [
+                "thumbnail"
+            ]
+        },
+        "geoparquet-items": {
+            "href": "abfs://items/esa-worldcover.parquet",
+            "title": "GeoParquet STAC items",
+            "description": "Snapshot of the collection's STAC items exported to GeoParquet format.",
+            "type": "application/x-parquet",
+            "roles": [
+                "stac-items"
+            ],
+            "table:storage_options": {
+                "account_name": "pcstacitems"
+            },
+            "msft:partition_info": {
+                "is_partitioned": false
+            }
+        }
+    },
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -180.0,
+                    -60.0,
+                    180.0,
+                    82.75
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2020-01-01T00:00:00Z",
+                    "2021-12-31T23:59:59Z"
+                ]
+            ]
+        }
+    },
+    "summaries": {
```

Comment:
From the [classification extension examples](https://github.com/stac-extensions/scientific/blob/25673d1777ba06a9b7b04819b8ed817757659924/examples/collection-summaries.json#L38-L40) it may be helpful to include the DOI in the summaries?

Reply:
There are two DOIs for this collection: one for the 2020 data and one for the 2021 data. We can't add two `sci:doi` fields to the collection, hence their existence in `links` only. These links are picked up and placed under a "DOI" heading on the dataset landing page in the PC.

There is an argument that perhaps we should include a `sci:doi` on each item, and then include the `sci:doi`s in the collection summaries. Half the Items would have one `sci:doi`, the other half the other one. But this seems very repetitive for something that can just be placed on the collection.

Marking as resolved. Feel free to re-open.